presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/APIFileHandler.java

```diff
@@ -102,9 +102,9 @@ public Response updateProperty(String path, String property, String value)
     {
         try {
             AgentFileUtils.updateProperty(Paths.get(baseDir.toString(), path), property, value);
-            LOGGER.debug("Updating property '%s' of file '%s' to '%s' in progress", property, path, value);
-            return Response.status(Status.ACCEPTED)
-                    .entity("Updating property of file in progress").build();
+            LOGGER.debug("Updating property '%s' of file '%s' to '%s'", property, path, value);
+            return Response.status(Status.OK)
```

Comment:
Synchronous. That is why we switch to OK instead of ACCEPTED. In the future, it will return different code for creation and updating.